### DISCUSSION TITLE
On the "Select the issue type" and "Is this issue blocking you" there is only 1 option (Github bug report)

### DISCUSSION THREAD

**Discussion by kenan238** [community/community#19257] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/73741117/175247322-a0a528e3-7f53-4ab5-8edd-a080b3548ce7.png)
![image](https://user-images.githubusercontent.com/73741117/175247381-15d43497-7c31-4624-9a1b-71cb5d9a62ef.png)

There is only 1 option
I tried on brave and chrome

**Comment by melvincwng** [comment]
@kenan238, I took a look at the bug report form, and ironically the form indeed seems to have an issue.

Instead, try contacting [Github Support](https://support.github.com/contact/account) over here.

Hope this clarifies, and please kindly 'Mark as answer' if you find it helpful.

Thanks 😊
